components/gemini/browser/gemini_json_parser.cc

```diff
@@ -137,7 +137,7 @@ bool GeminiJSONParser::GetDepositInfoFromJSON(
 bool GeminiJSONParser::GetOrderQuoteInfoFromJSON(
     const std::string& json, std::string *quote_id,
     std::string *quantity, std::string *fee, std::string *price,
-    std::string *error) {
+    std::string *total_price, std::string *error) {
   if (!quote_id || !quantity || !fee || !price) {
```

Comment:
Should this be updated to check `total_price` too now?

Reply:
Good catch - I'll update

Reply:
Fixed